src/legacy/core_plugins/vis_type_timeseries/server/validation_telemetry/validation_telemetry_service.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { APICaller, CoreSetup, Plugin } from 'kibana/server';
+import { get } from 'lodash';
+import { UsageCollectionSetup } from '../../../../../plugins/usage_collection/server';
+
+export interface ValidationTelemetryServiceSetup {
+  logFailedValidation: () => void;
```

Comment:
`validation_telemetry_service` returns async method. Should we wait result of internalRepository.incrementCounter or not? 

![image](https://user-images.githubusercontent.com/20072247/71176607-43adb200-227b-11ea-9ce6-3c4d386e5e41.png)

Reply:
That was done on purpose, when the telemetry request fails it shouldn't fail the user request because it's not an essential functionality